test/compliance/resources/db/basic/literals/basic.literals.number.js

```diff
@@ -16,31 +16,27 @@ module.exports = [
   },
   {
     decimal: '3.14153',
-    '=decimal': /^3.1415/
+    '=decimal': /3\.1415/
```

Comment:
I would keep the `^` as the value is only allowed to be longer in the fractionals. The `\` is a good catch.